low-level/framework/base/sspl_constants.py

```diff
@@ -109,7 +109,7 @@
 SSPL_SB_REQUEST = f"{SB_DATA_PATH}/sspl-sb-req.txt"
 
 # required only for system health
-HEALTH_UNDESIRED_VALS = ["N/A", ""]
+HEALTH_UNDESIRED_VALS = ["NA", "N/A", ""]
```

Comment:
We have undesired_values already  defined under RESOURCE_MAP in sspl_constants.py. Can we reuse it?

Reply:
It is not there. Please check https://github.com/Seagate/cortx-monitor/blob/main/low-level/framework/base/sspl_constants.py

Reply:
@mariyappanp please point exact code, so he can reuse it, if its available. I also remember vaguely about it

Reply:
@cdeshmukh @mariyappanp I saw this change in https://github.com/Seagate/cortx-monitor/pull/467 PR.
@mariyappanp you have introduced it. It is not merged yet.

Reply:
Yeah, sorry about it. I requested @cdeshmukh to merge it.

Reply:
@mariyappanp @cdeshmukh I think this ticket is also covering the #467 PR part. Here also we are doing "Not Available" and Not "Applicable" to every value where we can see "NA" or "N/A" or "".

Reply:
Agree. But have the HEALTH_UNDESIRED_VALS under const.RESOURCE_MAP for the context. Its not always for Health, but can be reusable across resource_map.

Reply:
Okay, @mariyappanp .
@cdeshmukh Please merge @mariyappanp PR if you are ok with it or merge this PR and then @mariyappanp make rebase.
Actually, this PR having a dependency on cortx-utils PR which is approved by Ujjwal https://github.com/Seagate/cortx-utils/pull/411